questions.md

```diff
@@ -4,26 +4,58 @@
 
 1. On the UCI cluster, the resource request "-pe openmp 64" refers to the number of processors requested.  Does that
    request mean that your commands will use multiple processors?
+   
+   #### It will use one or some or all processors depends on the job runs. "-pe openmp 64" will reserve 64 processors but if the job only uses several processors, the rest of the processors will not be in use.
+   
 2. In general, how do you know how many processors, how much RAM, how many files would be required/needed/written by the
    jobs you are running on the cluster?
+   
+   #### Use this command will show details about a specific job id
+   #### <pre><code> qstat -j (job ID) </pre></code> 
+   
 3. In order to be a "good citizen", you need to have some idea of much RAM your job requires.  In particular, you need
    to know the "peak" (i.e., maximum) RAM required at any point during execution.  Show an example of the shell command
    that you would use on a Linux machine to measure run time and peak ram usage of an arbitrary command, where the time/peak RAM values are written to a file.
+   #### Use this command to measure job running time and memory:
+   #### <pre><code>  /usr/bin/time/ samtools sort input_file > output_file </pre></code> 
```

Comment:
this is good but it is is also good to add option -o which allows an output file. env time -v -o MYOUTPUT CMD would do the trick and do a dump to a text file. this helps sometimes if you care about what goes in your stdout and stderr files. I like to keep it seperate as I usually do tail -n 1 MYSTDOUTFILE to see if I was able to run to completion and then sort my STDERRFILES using ls -lS jobname.ejobid* by size to see if the stderr produced larger output than normal. this typically means there where errors on the run. This doesn't work though when the programmers are sloppy and lazy and output stdout to stderr. :(
Also -v would be used if you wanted to know a lot more than just time and ram. for only selecting certain stats you can do -f "my options" and omit the -v